webclient/src/Hacksite.vue

```diff
@@ -35,4 +35,32 @@ hr {
   border: none;
   border-bottom: 1px solid #000;
 }
+.error {
```

Comment:
I like this idea! I believe there's a global css file that these should be put instead (static/hacksite.css)